statick_tool/plugins/tool/ruff_tool_plugin.py

```diff
@@ -26,23 +26,35 @@ def process_files(
     ) -> Optional[List[str]]:
         """Run tool and gather output."""
         flags: List[str] = []
-        flags += user_flags
 
+        # Ruff removed usage of plain ``ruff'' command in version 0.5.0. We need to make
+        # sure to use ``ruff check''.
+        try:
+            test_file: List[str] = []
+            if files:
+                test_file.append(files[0])
+            output = subprocess.check_output(
+                ["ruff"] + test_file, stderr=subprocess.STDOUT, universal_newlines=True
+            )
+        except subprocess.CalledProcessError as ex:
+            logging.debug("Exception in first check of ruff, need to add check verb")
+            logging.debug(ex)
+            flags.append("check")
+        except OSError as ex:
+            logging.warning("Couldn't find ruff executable! (%s)", ex)
+            return None
+
+        flags += user_flags
```

Comment:
did `check` exist before v0.5.0? It seems like it did from at least [v0.1.0](https://github.com/astral-sh/ruff/blob/main/CHANGELOG.md#010)

I think we should just always use `ruff check` instead of this complicated pre-check

Reply:
Good find. I agree. I'll update.

Reply:
we could add a >0.0.237 to the setup.py ruff requirement: https://github.com/sscpac/statick/blob/main/setup.py#L59